src/App.tsx

```diff
@@ -8,11 +9,22 @@ export const tabs: Tab[] = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+const App: React.FC = () => {
+  const [arrayTabs] = useState([...tabs]);
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+  const [selectedContent, setSelectedContent] = useState('Some text 1');
```

Comment:
don't create separate state for content, find content from selected tab by selectedTabId